src/background/index.ts

```diff
@@ -198,6 +198,17 @@ const messageListener = (request) => {
     })
   }
 
+  if (request.message === "delete_log") {
+    chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
+      const activeTab = tabs[0]
+
+      // Send a message to the active tab
+      agent.Logs.delete(request.log).then(response => {
+        chrome.tabs.sendMessage(activeTab.id, {message: "delete_log_response", body: response });
+      })
```

Comment:
We'd also change this to only receive an id, like so:

```suggestion
      agent.Logs.delete(request.id).then(response => {
        chrome.tabs.sendMessage(activeTab.id, {message: "delete_log_response", body: response });
      })
```